packages/vite-node/src/source-map.ts

```diff
@@ -63,7 +63,8 @@ export function withInlineSourcemap(
 
   // If the first line is not present on source maps, add simple 1:1 mapping ([0,0,0,0], [1,0,0,0])
   // so that debuggers can be set to break on first line
-  if (map.mappings.startsWith(';')) {
+  // This seems to affect coverage, so this is injected only for Vitest cli `--inspect-brk` usage.
+  if (map.mappings.startsWith(';') && process.argv.includes('--inspect-brk')) {
```

Comment:
It's also possible for this to come from configuration file, not just argv. Maybe we could do this with Vite plugin?

Reply:
Hmm, I haven't thought about that, but looks difficult. Probably it would be also nice if we can target injection to only test files since that's what inspector needs.